dockerfiles/che/entrypoint.sh

```diff
@@ -306,12 +306,11 @@ add_cert_to_truststore() {
   echo "$1" > $SELF_SIGNED_CERT
 
   # make sure that owner has permissions to write and other groups have permissions to read
-  # note that this might not work when running as anyuid OpenShift user, which is why we're forcing a true if it fails
-  chmod 644 $JAVA_TRUST_STORE || true
+  chmod 644 "$JAVA_TRUST_STORE"
```

Comment:
in case file is owned by root, this should be 777 (for anyuid login on openshift). In fact in CRW, if the file isn't already set to 777, this will fail.